Meetings/2021-09-07.md

```diff
@@ -2,33 +2,99 @@
 - Zoom Link: https://cuboulder.zoom.us/j/346594091
 
 ### Attending
--
+- J Luara
+- Jesse Mapel
+- Victor Silva
+- Michael Aye
+- Andrew Annex
+- Ross Beyer
 
 ## Agenda / Notes
 - Action Items from last time:
   - Jesse: Investigate other options for "Top Level Projects" terminology.
-  Any final updates from PDW21 (Jesse)?
+    - Punted
+  - Jesse: Any final updates from PDW21?
+    - No updates from PDW2021
 
 
 - Report from Top-Level Projects:
   - PlanetaryPy
+    - Lots of interest at the PDW2021
+      - Christian, from NAU, wrote a Jupyter book about getting into Python
```

Comment:
I'm uncomfortable including the specificity of people's names at this level in a report from a sub-group.  Can we just convert this item about Christian and the bullet below about Benoit to be: "Some additional attendees to PlanetaryPy TC meetings after PDW21."

Reply:
I've re-organized this section so that it makes a little more sense and scrubbed names.